src/trust_is_risk.js

```diff
@@ -14,10 +14,46 @@ var TrustDB = require("./trust_db");
 var DirectTrust = require("./direct_trust");
 
 class TrustIsRisk {
-  node : bcoin$FullNode
+  node : (bcoin$FullNode | bcoin$SPVNode)
```

Comment:
Check if you can type this with the superclass

Reply:
Done